### PR TITLE
aTask3a - Update Model to accommodate dropdown with query to select unique cohorts

### PR DESCRIPTION
**Pull request by bradleypurdue**
https://trello.com/c/L48gEbIH/227-3a-update-model-to-accommodate-dropdown-with-query-to-select-unique-cohorts

### CODE REVIEW

**Comment by amandahrdmn** [comment]
merging into master?

**Comment by bradleypurdue** [comment]
> merging into master?

whoops!

**Comment by turnerror** [comment]
task3b is basically in this too since we update the controller so we could test it actually worked, just working on the front end bit now.

File: src/Models/ApplicantModel.php [hunk 1]
```diff
@@ -70,6 +70,7 @@ public function insertNewApplicantToDb(ApplicantEntityInterface $applicant)
      * Sorts the table via the input taken from the sorting arrows
      *
      * @param string $sortingQuery
+     * @param string $cohortId
      * @return array $results is the data retrieved.
```
Comment (amandahrdmn) [inline src/Models/ApplicantModel.php:74]: update docs?

File: src/Models/ApplicantModel.php [hunk 2]
```diff
@@ -150,4 +149,31 @@ public function deleteApplicant($id)
         $query->bindParam(':id', $id);
         return $query->execute();
     }
+
```
Comment (amandahrdmn) [inline src/Models/ApplicantModel.php:152]: docs?

File: src/Models/ApplicantModel.php [hunk 3]
```diff
@@ -78,36 +79,34 @@ public function getAllApplicants(string $sortingQuery)
...
-                $stmt .= '`date` DESC';
-                break;
+        return $query->fetchAll();
+    }
 
-            default:
-                $stmt .= '`dateTimeAdded` ASC';
-                break;
-        }
+    public function getAllApplicantsByCohort(string $sortingQuery, string $cohortId)
```
Comment (amandahrdmn) [inline src/Models/ApplicantModel.php:94]: docs?

**Pull request merged**